front/src/modules/pathfinding/components/Itinerary/ModalSuggestedVias.tsx

```diff
@@ -9,21 +9,24 @@ import ModalBodySNCF from 'common/BootstrapSNCF/ModalSNCF/ModalBodySNCF';
 import ModalFooterSNCF from 'common/BootstrapSNCF/ModalSNCF/ModalFooterSNCF';
 import ModalHeaderSNCF from 'common/BootstrapSNCF/ModalSNCF/ModalHeaderSNCF';
 import { useOsrdConfActions, useOsrdConfSelectors } from 'common/osrdContext';
-import { isVia } from 'modules/pathfinding/utils';
+import { isVia, matchPathStepAndOp } from 'modules/pathfinding/utils';
 import type { SuggestedOP } from 'modules/trainschedule/components/ManageTrainSchedule/types';
+import type { PathStep } from 'reducers/osrdconf/types';
 import { useAppDispatch } from 'store';
 import { formatUicToCi } from 'utils/strings';
 
 type ModalSuggestedViasProps = {
   suggestedVias: SuggestedOP[];
+  launchPathfinding: (pathSteps: (PathStep | null)[]) => void;
 };
 
-const ModalSuggestedVias = ({ suggestedVias }: ModalSuggestedViasProps) => {
-  const { upsertViaFromSuggestedOP, clearVias, removeVia } = useOsrdConfActions();
-  const { getVias, getDestination } = useOsrdConfSelectors();
+const ModalSuggestedVias = ({ suggestedVias, launchPathfinding }: ModalSuggestedViasProps) => {
+  const { upsertViaFromSuggestedOP } = useOsrdConfActions();
+  const { getVias, getDestination, getPathSteps } = useOsrdConfSelectors();
   const dispatch = useAppDispatch();
   const vias = useSelector(getVias());
   const destination = useSelector(getDestination);
+  const pathSteps = useSelector(getPathSteps);
```

Comment:
We could add this here so we don't have to deal with `null` `PathStep` everywhere inside.
```suggestion
  const pathSteps = useSelector(getPathSteps) as PathStep[]; // this modal can't be open if we miss origin or destination
```

Reply:
You're right, done :+1: